### PR TITLE
Allow unarchiving projects.

### PR DESCRIPTION
**Pull request by cadeParade**
## Description
https://percy-io.slack.com/archives/C9BMMUPND/p1580143751000300

## Reviewing this PR

_**Is Covered by Automated Tests?: YES**_

### CODE REVIEW

File: app/components/projects/enabled-toggle.js [hunk 1]
```diff
@@ -1,21 +1,12 @@
...
-      project.save();
-    },
-    disable() {
-      let project = this.project;
-      project.set('isEnabled', false);
-      project.save();
-    },
-  },
+  toggleArchiveState: task(function*() {
+    const newState = this.project.isEnabled ? false : true;
```
Comment (unicornzero) [inline app/components/projects/enabled-toggle.js]: I'm curious, does `const newState = !this.project.isEnabled;` work, too?

File: app/templates/components/projects/enabled-toggle.hbs [hunk 2]
```diff
@@ -1,5 +1,12 @@
-{{#if this.project.isEnabled}}
-  <button type="button" class="btn btn-primary shadow-purple-lg mb-0" {{action "disable"}}>Archive Project</button>
-{{else}}
-  <button type="button" class="btn btn-primary shadow-purple-lg mb-0" {{action "enable"}}>Enable Project</button>
-{{/if}}
+<PercyBtn
```
Comment (unicornzero) [inline app/templates/components/projects/enabled-toggle.hbs]: <3

File: app/templates/organization/project/settings/index.hbs [hunk 3]
```diff
@@ -74,10 +74,15 @@
         </div>
       {{/unless}}
 
+    {{/if}}
+  {{/if}}
+
+  {{#let (variation 'only-admins-edit-settings') as |isFlagEnabled|}}
```
Comment (unicornzero) [inline app/templates/organization/project/settings/index.hbs]: TIL

Comment (unicornzero) [reply app/templates/organization/project/settings/index.hbs]: (didn't know about `let` passing the param in. very cool)

File: tests/acceptance/project-test.js [hunk 4]
```diff
@@ -386,6 +388,27 @@ describe('Acceptance: Project', function() {
...
+          });
+          expect(lookupProject().isEnabled).to.equal(true);
+
+          await ProjectSettingsPage.toggleArchiveButton.click();
+          expect(lookupProject().isEnabled).to.equal(false);
+          await percySnapshot(this.test);
+
+          await ProjectSettingsPage.toggleArchiveButton.click();
+          expect(lookupProject().isEnabled).to.equal(true);
+        });
```
Comment (unicornzero) [inline tests/acceptance/project-test.js]: Would there be any value in adding another `await percySnapshot(this.test);` here?

File: app/components/projects/enabled-toggle.js (hunk 1 above)
Comment (cadeParade) [reply app/components/projects/enabled-toggle.js]: duhhh i'm a dumb dumb

File: tests/acceptance/project-test.js (hunk 4 above)
Comment (cadeParade) [reply tests/acceptance/project-test.js]: well, we've got lots of snapshots of what the project settings page looks like, so I'm not sure it is super valuable to have another one.

**Comment by cadeParade** [comment]
Oh dear, didn't merge this yet 🤦‍♀

**Pull request merged**